coverage_comment/template.py

```diff
@@ -45,6 +43,7 @@ def get_comment_markdown(
     previous_coverage_rate: decimal.Decimal | None,
     base_template: str,
     custom_template: str | None = None,
+    marker: str = "<!-- This comment was produced by python-coverage-comment-action -->",
```

Comment:
I guess this might be a mandatory parameter, to avoid hardcode the default value in multiple places more than necessary